### DISCUSSION TITLE
Order of samples in the VCF output

### DISCUSSION THREAD

**Discussion by DaGaMs** [luntergroup/octopus#227] [Q&A] [answered]
Hi,

I'll put this here cause it's not really a bug report, more of a question. I'm really puzzled by the fact that I ran Octopus for somatic calling one some data a while back and the VCF file reported the INFO fields in the order `tumour` `normal`. However, I now ran octopus (v7) on some other data, but with the same order of command line, and I get `normal` `tumour`. What determines the order in which samples are reported in the VCF? How to ensure this is consistent between runs?

TIA,
Ben

**Comment by DaGaMs** [comment, marked as answer]
Duh... It's alphabetical, isn't it?

**Reply by dancooke** [reply]
Hi Ben. Indeed - lexicographical.
